communicator/ssh/provisioner.go

```diff
@@ -51,6 +54,9 @@ type connectionInfo struct {
 	ScriptPath string        `mapstructure:"script_path"`
 	TimeoutVal time.Duration `mapstructure:"-"`
 
+	KeepAliveInterval    string        `mapstructure:"keep_alive_interval"`
```

Comment:
This isn't added to config validation, so it can't actually be used from a configuration.